starter/package.json

```diff
@@ -130,8 +130,9 @@
     "@angular/platform-browser-dynamic": "~7.2.2",
     "@angular/platform-server": "~7.2.2",
     "@angular/router": "~7.2.2",
-    "@nationalbankbelgium/stark-core": "latest",
-    "@nationalbankbelgium/stark-ui": "latest",
+    "@nationalbankbelgium/code-style": "^1.0.1",
```

Comment:
Same here: I'm wondering if `code-style` is a dependency or a dev-dependency... what do you think?